Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -419,6 +419,40 @@ public let CodeGenerators: [CodeGenerator] = [
         b.destruct(arr, selecting: indices, into: candidates, hasRestElement: probability(0.2))
     },
 
+    CodeGenerator("DestructObjectGenerator", input: .object()) { b, obj in
+        var properties = Set<String>()
+        for _ in 0..<Int.random(in: 0...b.type(of: obj).numProperties) {
```

Comment:
I'm not sure this works well in practice. I think we might have some objects with a lot's of properties (e.g. Functions or the `Number` constructor) and then we'll probably have lot's of objects without properties (e.g. because the AI can't properly track them).
How about we just again make a loop with 1-5 iterations, then include a `randomProperty()` if available (and if we don't already have it), otherwise just a random property name?